danielMunteanu/src/main/java/Address.java

```diff
@@ -0,0 +1,26 @@
+public class Address {
+    public String country;
+    public String city;
+    public String street;
+
+    public Address(String country, String city, String street) {
+        this.country = country;
+        this.city = city;
+        this.street = street;
+    }
+
+    public String getFullAddress() {
+        return "Full address is" + " " + "Country:" + this.country + " " + "City: " + this.city + " " + "City:" + this.street;
+
+
+    }
+
+    @Override
+    public String toString() {
+        return "Address{" +
+                "country='" + country + '\'' +
+                ", city='" + city + '\'' +
+                ", street='" + street + '\'' +
+                '}';
+    }
```

Comment:
What is the difference between getFullAddress() and toString() method?

Reply:
these methods are basically doing the same, toString method will return all fields in string format